ngrinder-controller/src/test/java/org/ngrinder/perftest/service/PerfTestServiceTest.java

```diff
@@ -86,11 +87,11 @@ public void testGetTestListAll() {
 
 		for (PerfTest test : list) {
 			long systemTimeMills = System.currentTimeMillis();
-			test.setStartTime(new Date(systemTimeMills));
+			test.setStartTime(ofEpochSecond(systemTimeMills / 1000));
```

Comment:
Isn't it better to use Instant.now() here?

Reply:
Fixed